### ISSUE TITLE
Stable images are still based on alpine 3.17

### ISSUE THREAD

**Issue by orgads** [nginxinc/docker-nginx#847]
### Describe the bug

Looks like the base image was updated long ago in 7c10a4a1c2, but the official images are still based on 3.17.

### To reproduce

https://hub.docker.com/_/nginx/tags?page=1&name=1.24.0-alpine

### Expected behavior

It should be based on alpine 3.18.

**Comment by thresheek** [comment]
We did not have a stable release for 1.24.x, and moving the same tag between different OS versions has been proven a bad idea.

We'll wait until there is 1.24.1 release to bump this.

**Comment by orgads** [comment]
Is 1.24.1 expected soon?

If not, can you maybe *add* images name alpine3.18, and leave the existing "alpine" named images based on 3.17?

**Comment by thresheek** [comment]
There is no ETA for 1.24.x version bump.

Having multiple per-OS versions supported would be an approach multiple official images take, indeed - I'm not sure it's super beneficial for nginx though, as we don't really use a lot of fancy new features of the OS itself.  And there is still some overhead in shipping the images.

What would be your usecase for stable and 3.18, where you can't use mainline? (see https://www.nginx.com/blog/nginx-1-18-1-19-released/#NGINX-Versioning-Explained for differences)

**Comment by orgads** [comment]
Alpine 3.17 is vulnerable to CVE-2022-48174.

**Comment by thresheek** [comment]
Alpine 3.17 is still supported for security fixes until 2024-11-22, so we can expect it to get the updated busybox.  When this happens, and a new 3.17 minor release is available and published to Docker Library, nginx alpine-based image will be automatically rebuilt and the issue will be resolved.

Other option is to migrate to mainline nginx images based on alpine.

**Comment by Kinqdos** [comment]
Thank you for your answer @thresheek, but isn‘t it possible to release the explicit tag (eg: 1.25.4-alpine3.19) while keeping the „base“ tag? Or would this lead to, too much confusion?

**Comment by thresheek** [comment]
Yeah, I think it will be beneficial if we provide multiple alpine-based tags simultaneously, e.g. for the last two releases.  We had similar requests before.

We probably want to extend that to multiple debian versions as well, but I don't remember from the top of my head if we had such requests.   I guess it will make sense to do it though, since following one approach for both variants kinda makes most sense rather than having different policies.

I'll work on preparing infrastructure for that.

**Comment by Kinqdos** [comment]
Sounds awesome thank you!
